hello-spring/src/main/java/org/folio/hello/SumController.java

```diff
@@ -0,0 +1,30 @@
+package org.folio.hello;
+
+import org.folio.hello.api.SumApi;
+import org.folio.hello.model.InputIntegers;
+import org.folio.hello.model.SumResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public final class SumController implements SumApi {
+
+  /** {@inheritDoc} */
+  @Override
+  public ResponseEntity<SumResponse> sumBodyGet(InputIntegers body) {
```

Comment:
Meant for this to be a POST, oops

```suggestion
  public ResponseEntity<SumResponse> sumBodyPost(InputIntegers body) {
```